clients/src/main/java/org/apache/kafka/clients/consumer/Consumer.java

```diff
@@ -59,24 +59,24 @@
     public ConsumerRecords<K, V> poll(long timeout);
 
     /**
-     * @see KafkaConsumer#commit(CommitType)
+     * @see KafkaConsumer#commit()
      */
-    public void commit(CommitType commitType);
+    public void commit();
 
     /**
-     * @see KafkaConsumer#commit(CommitType, ConsumerCommitCallback)
+     * @see KafkaConsumer#commit(OffsetCommitCallback)
```

Comment:
We changed ConsumerRebalanceCallback to ConsumerRebalanceListener. Can we standardize on that naming scheme for callbacks and call this OffsetCommitListener?